src/main/scala/org/constellation/domain/transaction/PendingTransactionsMemPool.scala

```diff
@@ -55,7 +55,7 @@ class PendingTransactionsMemPool[F[_]: Concurrent](
               }
               .map { last =>
                 val ordinal = last.ordinal
-                val left = txs.filter(_.transaction.lastTxRef.ordinal >= ordinal) // TODO: get rid of >=
+                val left = txs
```

Comment:
@buckysballs @marcinwadon `sortAndFilterForPull` filters transactions and does not allow transactions with a lower number to be retrieved, 
so if we have double spend txs, one transaction is taken to consensus and one remains in mempool
My question is how it should behave?

Reply:
Why have you removed the filter fn here?

Reply:
I wrote above